fronts-client/src/fixtures/shared.ts

```diff
@@ -591,11 +593,22 @@ const normalisedCollectonWithPreviously = {
 	excludedRegions: [],
 };
 
-const collectionConfig = {
+const collectionConfig: CollectionConfig = {
 	id: 'exampleCollection',
 	displayName: 'Example Collection',
 	type: 'any',
-	groups: ['large', 'medium', 'small'],
```

Comment:
Existing containers have a concept of groups. Is the plan to move to groupsConfig for all containers rather than just for "beta" containers?

Reply:
ahh ignore the above, i see that we're supporting both here https://github.com/guardian/facia-scala-client/pull/345/files 👍

Reply:
am keeping groups for now as we'll need to support clients who haven't upgraded to the latest model 👍